lib/email_events/adapters/abstract/event_data.rb

```diff
@@ -0,0 +1,29 @@
+module EmailEvents::Adapters
+  module Abstract
+    class EventData
+      [:event_type, :event_timestamp, :recipient, :status_string, :smtp_status_code, :reason,
+       :smtp_message_id, :provider_message_id, :simplified_status, :raw_data].each do |pure_virtual_method|
+        define_method(pure_virtual_method) { raise "Not implemented" }
+      end
+
+      def simplified_status
+        # try to get a specific status based on the smtp status code; however, if the event doesn't have an smtp
+        # status code (eg. bounce events always do, but drop events only do sometimes), supply a generic one
+        return "Unable to send email to the address provided" if smtp_status_code.blank?
```

Comment:
This returns a string description while the method normally returns a shorthand symbol. Is that intentional?